doc/source/workflow/overview.md

```diff
@@ -0,0 +1,208 @@
+- [Service Orchestrator](./overview.html#service-orchestrator) for advanced inference graphs
+
+as well as integration with third-party systems:
+- Kubernetes Ingress integration with [Ambassador](https://www.getambassador.io/) and [Istio](https://istio.io/)
+- [Metrics](./overview.html#metrics-with-prometheus) with [Prometheus](https://prometheus.io/)
+- [Tracing](./overview.html#distributed-tracing-with-jaeger) with [Jaeger](https://www.jaegertracing.io/)
+- [Endpoint Documentation](./overview.html#endpoint-documentation) with [OpenApi](https://swagger.io/docs/specification/about/)
+
+Keep reading to learn more!
+
+
+![](../images/e2e-model-serving.svg)
+
+
+Usual e2e model serving with Seldon consists of few very important steps:
+1. *Data Scientist* prepare ML `model` using state of the art libraries (mlflow, dvc, xgboost, scikit-learn just to name a few).
+2. Trained model is uploaded to the central repository (e.g. S3 storage).
+3. *Software Engineer* prepare a `Model Server` which is uploaded as Docker image to the Image Registry.
+4. Deployment manifest (`SeldonDeployment` CRD) is created and applied to the Kubernetes cluster.
+5. Seldon Core `Operator` creates all required Kubernetes resources.
+6. Inference requests are served by Seldon Core `Orchestrator`.
+7. Metrics and tracing data is written to the third party systems.
+
+## Model Servers
+
+With Seldon Core you can build two type of servers: reusable and non-reusable ones.
+Each of these are useful depending on the context and the actual use case.
+
+- **Reusable Model Servers**: Often referred to as prepackaged model servers.
+  Allow to deploy a family of similar models without the need to build a new server each time.
+  They often fetch models from a central repository (like your company's S3 storage)
+- **Non-Reusable Model Servers**: Specialised server meant to serve a single model.
+  Does not require the central repository but requires a build of a new image for every model.
+
+
+![](../images/model-servers.svg)
+
+Read more about our pre-packaged `Model Servers` on their dedicated documentation pages:
+- [MLflow Server](../servers/mlflow.html)
+- [SKLearn Server](../servers/sklearn.html)
+- [Tensorflow Server](../servers/tensorflow.html)
+- [XGBoost Server](../servers/xgboost.html)
+
+Read how to build your own pre-packaged model server [here](../servers/custom.html).
+
+## Language Wrappers
+
+Language wrappers allows Seldon Core users to build reusable and non-reusable model servers.
+As you will see, the whole process is very simple and requires user to only define logic that
+loads models and perform inference prediction as well as required runtime dependencies.
+
+![](../images/language-wrappers-1.svg)
+
+
+Model loading and inference logic is defined in `Model.py` file:
+```python
+class Model:
+  def __init__(self):
+    # custom logic that prepare model
+    self._model = ...
```

Comment:
here you should specify that the model is loaded from local filestore baked inside the image instead of downloading it dynamically

Reply:
It may be worth also adding either a simple example or pseudo code of the pattern that reusable servers actually use, which is basically downloading the model on startup, which is the main difference between both

Reply:
As in reusable model servers take different arguments to the `init` function and work together with the InitContainers?

Reply:
Oh right, yeah that's one of the main things, but mainly the main difference boils down to reusable servers dynamically loading the image and non-reusable baking it in (which is reflected on the python wrapper, here you show `model = ...` but it could be more explicit such as `model = your_loader("local/file/path")` or somtehing similar

Reply:
How about having one snippet with description?
```
```python
class Model:
  def __init__(self):
    # custom logic that prepares model
    # Reusable model servers: your_loader downloads model from remote repository
    # Non-Reusable servers: your_loader loads model from a file embedded in the image
    self._model = your_loader(...)

  def predict(self, features, names=[], meta=[]):
    # custom inference logic
    return self._model
```

Reply:
Yeah this seems reasonable, although it may be worth making it explicit, this is not much code, so it may be worth just adding an exmaple on each, and just making it clear that with non-reusable it's local baked in image, and reusable is download from external